src/test/java/org/jpeek/metrics/Lcom4Test.java

```diff
@@ -84,7 +84,7 @@
      *  After implementing, remove the @Ignore.
      */
     @Test
-    @Ignore
+    @Disabled
```

Comment:
@HDouss Do you know why this test is ignored/disabled ????

Reply:
@fanifieiev It should be related to the puzzle at L77, no relation to this issue in particular.